packages/io/svg-deserializer/shapesMapGeometry.js

```diff
@@ -4,12 +4,12 @@ const { svg2cagX, svg2cagY, cagLengthX, cagLengthY, cagLengthP, reflect } = requ
 // const { cssPxUnit } = require('./constants')
 
 const shapesMapGeometry = (obj, objectify, params) => {
-  const { svgUnitsPmm, svgUnitsX, svgUnitsY, svgUnitsV, svgGroups, target } = params
+  const { svgUnitsPmm, svgUnitsX, svgUnitsY, svgUnitsV, svgGroups, target, segments } = params
 
   const types = {
-    group: (obj) => objectify({ target }, obj),
+    group: (obj) => objectify({ target, segments }, obj),
 
-    rect: (obj, svgUnitsPmm, svgUnitsX, svgUnitsY) => {
+    rect: (obj, svgUnitsPmm, svgUnitsX, svgUnitsY, svgUnitsV, svgGroups, segments) => {
```

Comment:
UG UG... not ideal

Reply:
Yeah... This is not great. Given how much pick and choose there is, it would be nice to have that as an object you can destructure but I imagine we'd have to match that across all deserialisers? Maybe a refactor for another time?

Reply:
Yeah. Another time. ;)